packages/grid/_modules_/grid/components/toolbar/GridToolbarExport.tsx

```diff
@@ -23,116 +14,72 @@ interface GridExportDisplayOptions {
+  const apiRef = useGridApiContext();
+  const { hideMenu, options } = props;
 
-    const handleMenuOpen = (event) => {
-      setOpen(true);
-      onClick?.(event);
-    };
-    const handleMenuClose = () => setOpen(false);
-    const handleExport = (option: GridExportMenuItemsParams) => () => {
-      switch (option.format) {
-        case 'csv':
-          apiRef.current.exportDataAsCsv(option.formatOptions);
-          break;
-        case 'print':
-          apiRef.current.exportDataAsPrint(option.formatOptions);
-          break;
-        default:
-          break;
-      }
+  if (options?.disableToolbarButton) {
+    return null;
+  }
+  return (
+    <MenuItem
+      onClick={() => {
+        apiRef.current.exportDataAsPrint(options);
+        hideMenu?.();
+      }}
+    >
+      {apiRef.current.getLocaleText('toolbarExportPrint')}
+    </MenuItem>
+  );
+};
 
-      setOpen(false);
-    };
-
-    const handleListKeyDown = (event: React.KeyboardEvent) => {
-      if (isTabKey(event.key)) {
-        event.preventDefault();
-      }
-      if (isHideMenuKey(event.key)) {
-        handleMenuClose();
-      }
-    };
+const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportProps>(
+  function GridToolbarExport(props, ref) {
+    const { csvOptions = {}, printOptions = {}, ...other } = props;
 
     if (csvOptions?.disableToolbarButton && printOptions?.disableToolbarButton) {
       return null;
     }
 
     return (
-      <React.Fragment>
-        <rootProps.components.BaseButton
-          ref={handleRef}
-          color="primary"
-          size="small"
-          startIcon={<rootProps.components.ExportIcon />}
-          aria-expanded={open ? 'true' : undefined}
-          aria-label={apiRef.current.getLocaleText('toolbarExportLabel')}
-          aria-haspopup="menu"
-          aria-labelledby={menuId}
-          id={buttonId}
-          {...other}
-          onClick={handleMenuOpen}
-          {...rootProps.componentsProps?.baseButton}
-        >
-          {apiRef.current.getLocaleText('toolbarExport')}
-        </rootProps.components.BaseButton>
-        <GridMenu
-          open={open}
-          target={buttonRef.current}
-          onClickAway={handleMenuClose}
-          position="bottom-start"
-        >
-          <MenuList
-            id={menuId}
-            className={gridClasses.menuList}
-            aria-labelledby={buttonId}
-            onKeyDown={handleListKeyDown}
-            autoFocusItem={open}
-          >
-            {exportOptions.map((option, index) =>
-              option.formatOptions?.disableToolbarButton ? null : (
-                <MenuItem key={index} onClick={handleExport(option)}>
-                  {option.label}
-                </MenuItem>
-              ),
-            )}
-          </MenuList>
-        </GridMenu>
-      </React.Fragment>
+      <GridToolbarExportContainer {...other} ref={ref}>
```

Comment:
If you pass children, they are not used.

What is the wanted behavior ? Do we concatenate them with the default or do they override the default ?